PROTOCOL.md

```diff
@@ -142,6 +142,8 @@ represented by a space), and the rest of the string being their username.
 
 > We received an HTML message, which should be sanitized and displayed
 > directly in the room's log.
+> Unusable tags: `<iframe>`, `<script>`, `<base>`, `<meta>`, `<canvas>`, `<link>`, and `<head>`.
+> Custom tags: `<psicon [pokemon or item]=[source]>`, `<username [name?]=[source name]>`
```

Comment:
type and category are also supported by `psicon`.

Also this seems very unspecific.

Also this is a run-on sentence. Remember that Markdown doesn't automatically insert line-breaks every time you press Enter.

Reply:
![image](https://user-images.githubusercontent.com/551184/96610948-03f79280-12b1-11eb-9423-a78a2b523858.png)